src/providers/windshaft.js

```diff
@@ -1,13 +1,76 @@
 var d3 = require("d3");
+var _ = require("underscore");
+var Providers = require("./");
 
 function WindshaftProvider(options) {
-	this.sql_api_template = options.sql_api_template || 'http://{user}.cartodb.com';
+	this.tiler_template = options.tiler_template || 'http://{user}.cartodb.com';
 	this.user = options.user;
 	this.table = options.table;
 	this.format = options.format;
+	this.options = options;
 	this.tileCache = {};
+	this.initialize();
 }
 
 WindshaftProvider.prototype = {
+	initialize: function(){
```

Comment:
Why don't we have this in the constructor? Is there any other way to use initialize apart from being called from the constructor?